### PR TITLE
Merge 2.21 code freeze to develop

### PR DESCRIPTION
**Pull request by oguzkocer**

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+-----
+
+
 2.21
 -----
 * [Internal] Added login alert for unverified accounts [#1449](https://github.com/Automattic/simplenote-android/pull/1449)
 * Fixed scroll position of a note with matching keyword in not persistent [#1454](https://github.com/Automattic/simplenote-android/pull/1454)
 * Updated icons [#1458](https://github.com/Automattic/simplenote-android/pull/1458)
-* Fixed editor's background color in landscape mode with dark theme [#1457](https://github.com/Automattic/simplenote-android/pull/1457)
+* Fixed editor's background color in landscape mode for dark theme [#1457](https://github.com/Automattic/simplenote-android/pull/1457)
```
Comment (oguzkocer) [inline RELEASE-NOTES.txt:10]: @danilo04 I've made a very small change to this release note. Hope that's OK with you.

Comment (danilo04) [reply RELEASE-NOTES.txt:10]: 💯 . Thanks!

File: Simplenote/build.gradle [hunk 2]
```diff
@@ -102,7 +102,7 @@ dependencies {
     // Fastlane screengrab for screenshots automation
     androidTestImplementation 'tools.fastlane:screengrab:2.0.0'
     // Automattic and WordPress dependencies
-    implementation 'com.automattic:simperium:develop-78e8200288a4d0e75937720cbd0e503ddb04f035'
+    implementation 'com.automattic:simperium:v1.0.0'
```
Comment (mokagio) [inline Simplenote/build.gradle:105]: 🎉

**Review by mokagio — APPROVED** [review]
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] Version updates in `build.gradle`
- [x] `strings.xml` updates via automation
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version

**Pull request merged**